sport/app/AppLoader.scala

```diff
@@ -16,6 +15,7 @@ import football.controllers.{FootballControllers, HealthCheck}
 import http.{CommonFilters, CorsHttpErrorHandler}
 import jobs.CricketStatsJob
 import model.ApplicationIdentity
+import org.apache.pekko.stream.{Materializer => PekkoMaterializer}
```

Comment:
It looks like you've migrated to `PekkoMaterializer`, right? Just noting this because in the PR description it's mentioned you've kept `akka.stream.Materializer`.